test/unit/proxy_test.rb

```diff
@@ -483,8 +483,13 @@ def test_apicast_configuration_driven
                               api_backend: 'http://example.com',
                               api_test_path: '/path',
                               apicast_configuration_driven: false)
-    ::ApicastV1DeploymentService.any_instance.expects(:deploy).with(proxy).returns(true).never
+    ::ApicastV1DeploymentService.any_instance.expects(:call).returns(true)
```

Comment:
This will fail if you removed it no?
Replace it with the equivalent of v2 if it does not exist already

Reply:
since you are using Apicast, change `apicast_configuration_driven: false` to `true`